src/main/java/uk/gov/ons/census/casesvc/service/ReceiptProcessor.java

```diff
@@ -58,12 +58,16 @@ public void processReceipt(ResponseManagementEvent receiptEvent) {
     }
 
     UacQidLink uacQidLink = uacQidLinkOpt.get();
-    Case caze = uacQidLink.getCaze();
+    uacQidLink.setActive(false);
+    uacQidLinkRepository.saveAndFlush(uacQidLink);
 
-    uacProcessor.emitUacUpdatedEvent(uacQidLink, caze, false);
+    Case caze = uacQidLink.getCaze();
     caze.setReceiptReceived(true);
     caseRepository.saveAndFlush(caze);
+
+    uacProcessor.emitUacUpdatedEvent(uacQidLink, caze, uacQidLink.isActive());
```

Comment:
why do you need to pass in the isActive if you're already passing in the uacQidLink (which contains the isActive)?

Reply:
because basically, it's a one line fix, didn't want to start refactoring to that level - emitUacUpdatedEvent() is overloaded which always passes in "true" - needs a cleanup but not now